src/translate.rs

```diff
@@ -921,9 +921,19 @@ where T: GlibPtrDefault + ToGlibPtr<'a, <T as GlibPtrDefault>::GlibType> {
 
 pub struct Array(*mut glib_sys::GArray);
 
+impl Array {
+    fn new(zero_terminated: bool, clear: bool, element_size: usize) -> Self {
```

Comment:
If you store `T` in `Array` directly (through `PhantomData`), `element_size` becomes useless as argument.